README.md

```diff
@@ -1,16 +1,33 @@
-# CovidO: The CoronaVirus Disease Ontology (CovidO)
+# The CoronaVirus Disease Ontology (CovidO)
 
-# What is the CovidO ?
-The CovidO ontology specifies a foundational metadata model set for real-world multimodal CoronaVirus. The CovidO provides an Ontological Framework for Representation and annotation of COVID-19 Information. This is an extension of CODO (COviD-19 Ontology) in real-world multimodal dimensions. 
+# What is CovidO?

```

Comment:
```suggestion
## What is CovidO?
```

Reply:
There should only be one top-heading (`#`) in any given `.md` file.  Subheads should have 2 (`##`), 3 (`###`), or more hashes.